styles.css

```diff
@@ -0,0 +1,98 @@
+
+body{
+  background-color: #E8EBEE;
+  margin:auto;
+
+}
+
+header {
+    background-color: #C2DBF4;
+    color: white;
+    text-align: center;
+    height: 100px;
+    overflow: auto;
+    border-bottom: 2px inset;
+}
+
+.wrapper{width: 800px;
+  margin:auto;
+  border-right: 2px groove;
+  border-left:2px groove;
+
+}
+main{
+  background-color:#94B8FA;
+  text-align: left;
+  width:460px;
+  padding:9px;
+  display: inline-block;
+  border-right: 1px groove;
+}
+aside {
+  background-color:#A2CFD5;
+  text-align: left;
+  width:300px;
+  padding:10px;
+  display: inline-block;
+  float: right;
+  height: 487px;
+}
+aside h3{text-align: center; font-size: 25px;}
```

Comment:
Try to keep these spaced out and indented like they are above as it makes the code more readable.